src/r2mm/downloading/BetterThunderstoreDownloader.ts

```diff
@@ -98,36 +98,39 @@ export default class BetterThunderstoreDownloader extends ThunderstoreDownloader
         const dependencies = await this.getDependenciesWithCorrectVersions(combo, modList);
         const allModsToDownload = [...dependencies, combo];
 
+        let modInProgressName = combo.getMod().getName();
         let downloadCount = 0;
-        const singleModProgressCallback = (progress: number, status: number, err: R2Error | null) => {
+        const singleModProgressCallback = (downloadProgress: number, status: number, err: R2Error | null) => {
             if (status === StatusEnum.FAILURE) {
                 throw err;
             }
 
-            let totalProgress: number;
+            let totalDownloadProgress: number;
             if (status === StatusEnum.PENDING) {
-                totalProgress = this.generateProgressPercentage(progress, downloadCount, allModsToDownload.length);
+                totalDownloadProgress = this.generateProgressPercentage(downloadProgress, downloadCount, allModsToDownload.length);
```

Comment:
Not having "download is 80%, extracting 20%" multiplier here like there's in `downloadImportedMods` leads to situation where downloading e.g. "TV Loader Anime Openings" shows download being completed and installation "waiting for download" for two minutes while the videos are extracted from the zip.